### ISSUE TITLE
draft-06 / draft-07 support?

### ISSUE THREAD

**Issue by handrews** [mafintosh/is-my-json-valid#160]
Hello from the JSON Schema spec project! Are you planning to implement any of the newer drafts in this project? We are reorganizing the lists on json-schema.org to highlight projects supporting (or working on supporting) draft-06 or later.

See http://json-schema.org/specification.html for info on the latest published drafts.

**Comment by LinusU** [comment]
We would love to be 🚀 

But currently, as far as I know, there is no one that is working on it.

Do you know how to best get an overview on what has changed since earlier versions? I would be happy to work against it 👏

**Comment by handrews** [comment]
Yes, we have migration notes on the specification page http://json-schema.org/specification.html#migrating-from-older-drafts

Feel free to ask questions here if there's anything unclear, or join us on the [slack channel](https://join.slack.com/t/json-schema/shared_invite/enQtMjk1NDcyNDI2NTAwLTcyYmYwMjdmMmUxNzZjYzIxNGU2YjdkNzdlOGZiNjIwNDI2M2Y3NmRkYjA4YmMwODMwYjgyOTFlNWZjZjAyNjg).
